Source/UI/BillingForm/BillingFormCell.swift

```diff
@@ -1,7 +1,6 @@
 import Foundation
 
 @frozen public enum BillingFormCell {
```

Comment:
Unrelated but this @frozen should go.

Could we have a chat about reviewing the behaviour of frozen and maybe making changes before this goes out?